scripts/procdockerstatsd

```diff
@@ -135,7 +136,13 @@ class ProcDockerStats(daemon_base.DaemonBase):
         return True
 
     def update_processstats_command(self):
-        data = self.run_command("ps -eo uid,pid,ppid,%mem,%cpu,stime,tty,time,cmd --sort -%cpu | head -1024")
+        cmd0 = ["ps", "-eo", "uid,pid,ppid,%mem,%cpu,stime,tty,time,cmd", "--sort", "-%cpu"]
+        cmd1 = ["head", "-1024"]
+        cmd = ' | '.join([' '.join(cmd0), ' '.join(cmd1)])
```

Comment:
>cmd [](http://example.com/codeflow?start=8&length=3)

You can move `cmd` to inside `if` block, so it will optimize performance a little bit.